source/includes/_amp.md

```diff
@@ -25,15 +24,15 @@ Add the following AMP imports to the `<head>` element of your page.
 <script async custom-element="amp-bind" src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"></script>
 ```
 
-### Step 2: add Tipser inline styles ###
+### Styles
 
-Add the following Tipser styles to the `<head>` element of your page.
+Add the following Tipser styles to the `<head>` element of your page. They are not mandatory but highly recommended.
```

Comment:
Not sure if we need this the newly added statement. AFAIR our AMP elements are not really usable without those styles so it's better if our customers just always includes them (but we can mention that they are free to tweak them if they want).

Reply:
Yes, ok.